### PR TITLE
Add exercises on accessing object properties

### PR DESCRIPTION
**Pull request by cdersky**
# Summary

 using dot notion and bracket notation with strings and variables. The Objects Access slide needs to be updated with a variable example. And a link to this exercise needs to be added to the objectsdom.html slide deck. This is in response to issue #228.

Fixes issue #228 

Here's a description of changes:
- Added 3 new exercises

### CODE REVIEW

**Comment by cdersky** [comment]
@brendajin These new exercises address some concerns we had this weekend. They can be expanded to make more exercises on access in the later material in the same slide deck (objectsdom.html). I just wanted to make sure that this is the type of thing that is appropriate to add before creating new exercises.

**Comment by verythorough** [comment]
Ooh, cool.  I'll be teaching these lessons later this week or early next week.  I'll take a look at the exercises tonight or tomorrow.

File: javascript/exercises/objectIntro.html [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+    <link href="../../common/css/bootstrap.css" rel="stylesheet">
+  </head>
+ <body>
+<div class="container">
+<h2>Exercises: Objects</h2>
+
+
+<h3 class="page-header">Object Property Access</h3>
+
+<p><em>Major US Cites:  List all major cities two times, in the majorCities object, using <code>console.log</code> statements. The first time, list all the major cities using dot notation. The second time, list all the major cities using bracket notation. </em></p>
```
Comment (brendajin) [inline javascript/exercises/objectIntro.html]: I like this exercise! I think we can do more of a ramp up. What do you think of this: "Here is an object that includes major cities. Use `console.log()` to print properties of the object. Print all the properties using dot notation, then bracket notation"

File: javascript/exercises/objectIntro.html [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+console.log(majorCities["California"]); // "Los Angeles"
+console.log(majorCities["Illinois"]); // "Chicago"
+console.log(majorCities["Texas"]); // "Houston"
+console.log(majorCities["Pennsylvania"]); // "Philadelphia"
+console.log(majorCities["Arizona"]); // "Phoenix"
+</pre>
+
+<h3 class="page-header">Object Property Access</h3>
+
+<p><em>Store each city name in a separate variable. Use both bracket and dot notation. Use <code>console.log</code> statements to check your work. </em></p>
```
Comment (brendajin) [inline javascript/exercises/objectIntro.html]: >Use new variables to store each city name, using both bracket and dot notation. Use `console.log()` with your new variables to check your work.

File: javascript/exercises/objectIntro.html [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+  console.log(city2); // "Los Angeles"
+  console.log(city3); // "Chicago"
+  console.log(city4); // "Houston"
+  console.log(city5); // "Philadelphia"
+  console.log(city6); // "Phoenix"  
+</pre>
+
+ <h3 class="page-header">Object Property Access With Variables</h3>
+
+<p><em>In the first example, you used strings with bracket notation to access property values. In this exercise use <i>variable</i> names to access property values.</em></p>
```
Comment (brendajin) [inline javascript/exercises/objectIntro.html]: Maybe add a "see example below:"? I really like that this one has the starter example.

File: javascript/exercises/objectModify.html [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+    <link href="../../common/css/bootstrap.css" rel="stylesheet">
+  </head>
+ <body>
+<div class="container">
+<h2>Exercises: Objects continued</h2>
+
+
+<h3 class="page-header">Overwrite Object Properties</h3>
+
+<p><em>Change the cities listed in the <code>UnitedStatesCities</code> object to the each state's capitol. Use <code>console.log</code> statements to check your work. </em></p>
```
Comment (brendajin) [inline javascript/exercises/objectModify.html]: remove extra "the"

File: javascript/exercises/objectModify.html [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+console.log(UnitedStatesCities.California); // "Los Angeles"
+console.log(UnitedStatesCities.Illinois); // "Chicago"
+console.log(UnitedStatesCities["Texas"]); // "Houston"
+console.log(UnitedStatesCities["Pennsylvania"]); // "Philadelphia"
+console.log(UnitedStatesCities["Arizona"]); // "Phoenix"
+</pre>
+
+<h3 class="page-header">Add Object Values and Properties</h3>
+
+<p><em>Add five more states and their capitol cities to the <code>UnitedStatesCities</code> object. Use <code>console.log</code> statements to check your work. </em></p>
```
Comment (brendajin) [inline javascript/exercises/objectModify.html]: love this

File: javascript/exercises/objectModify.html [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+  UnitedStatesCities["Alabama"] = "Montgomery";
+  UnitedStatesCities["Alaska"] = "Juneau";
+  UnitedStatesCities["West Virgina"] = "Charleston";
+  UnitedStatesCities.Wisconsin = "Madison";
+  UnitedStatesCities.Wyoming = "Cheyenne";
+  
+  console.log(UnitedStatesCities)
+</pre>
+
+ <h3 class="page-header">Delete Object Values and Properties</h3>
```
Comment (brendajin) [inline javascript/exercises/objectModify.html]: love this too!

**Comment by brendajin** [comment]
Overall, looks super awesome! Thank you for adding these exercises. I suggested some minor edits and copy changes, but otherwise I can't wait to merge this in.

#250

**Pull request merged**